### PR TITLE
Clean up table types documentation config and include it in a group with display attributes.

### PR DESCRIPTION
**Pull request by ewencp**

### CODE REVIEW

**Comment by ewencp** [comment]
@confluentinc/connect Minor cleanup after #37

File: src/main/java/io/confluent/connect/jdbc/JdbcSourceConnectorConfig.java [hunk 1]
```diff
@@ -155,14 +155,16 @@
...
       "By default, the JDBC connector will only detect tables with type TABLE from the source Database. "
-      + "This config allows a command seperated list of table types to extract. Optons include:\n "
-      + "* TABLE "
-      + "* SYSTEM TABLE "
-      + "GLOBAL TEMPORARY "
-      + "* LOCAL TEMPORARY "
-      + "* ALIAS "
-      + "* SYNONYM "
-      + "In most cased it only makes sense to have either Table or View, depending on the usecase. ";
+      + "This config allows a command seperated list of table types to extract. Optons include:\n"
```
Comment (hachikuji) [inline src/main/java/io/confluent/connect/jdbc/JdbcSourceConnectorConfig.java]: Typos: "seperated" and "Optons"

File: src/main/java/io/confluent/connect/jdbc/JdbcSourceConnectorConfig.java [hunk 2]
```diff
@@ -155,14 +155,16 @@
...
+      + "* TABLE\n"
+      + "* VIEW\n"
+      + "* SYSTEM TABLE\n"
+      + "* GLOBAL TEMPORARY\n"
+      + "* LOCAL TEMPORARY\n"
+      + "* ALIAS\n"
+      + "* SYNONYM\n"
+      + "In most cased it only makes sense to have either Table or View, depending on the usecase.";
```
Comment (hachikuji) [inline src/main/java/io/confluent/connect/jdbc/JdbcSourceConnectorConfig.java]: Typo: "cased". Also, maybe just leave off "depending on the usecase"? Seems redundant.

**Comment by hachikuji** [comment]
LGTM

**Pull request merged**